docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials_full_akri/_index.md

```diff
@@ -0,0 +1,277 @@
+---
+type: docs
+title: "Discover ONVIF cameras with Akri on AKS Edge Essentials multi-node deployment"
+linkTitle: "Discover ONVIF cameras with Akri on AKS Edge Essentials multi-node deployment"
+weight: 3
+description: >
+---
+
+## Discover ONVIF cameras with Akri on AKS Edge Essentials multi-node deployment
+
+The following Jumpstart scenario will show how to create an AKS Edge Essentials full deployment with two VMs in Hyper-V nested virtualization in an Azure Windows Server VM, and connect the Hyper-V VMs and AKS Edge Essentials cluster to Azure Arc using [Azure Bicep](https://learn.microsoft.com/azure/azure-resource-manager/bicep/overview). Once Edge Essentials is deployed Akri is installed as a Kubernetes resource interface that exposes an IP mock camera as resources in the Edge Essentials cluster.
```

Comment:
```suggestion
The following Jumpstart scenario will show how to create an AKS Edge Essentials full deployment with two VMs in Hyper-V nested virtualization in an Azure Windows Server VM, and connect the Hyper-V VMs and AKS Edge Essentials cluster to Azure Arc using [Azure Bicep](https://learn.microsoft.com/azure/azure-resource-manager/bicep/overview). Once Edge Essentials is deployed [Akri](https://docs.akri.sh/) is installed as a Kubernetes resource interface that exposes an IP mock camera as resources in the Edge Essentials cluster.
```

Reply:
Also, maybe add some context on Akri and ONVIF to set the stage for the scenario

Reply:
Also add a link to give more context on ONVIF and in the first instance, don't use just the acronym. 

- Open Network Video Interface Forum
- https://wikipedia.org/wiki/ONVIF